civihr_default_theme/template.php

```diff
@@ -904,3 +916,30 @@ function _add_sub_menu_to_link(&$link) {
 
   return '<ul class="dropdown-menu">' . drupal_render($link['#below']) . '</ul>';
 }
+
+/**
+ * Sets the maximum viewport scale to 1.
+ * @NOTE this still allows users to pinch/zoom with 2 fingers.
+ * @NOTE this function expects that the Meta Viewport tag is already set.
+ *
+ * @param object $head_elements
+ */
+function _set_maximum_scale_to_viewport_meta_tag(&$head_elements) {
+  // Get the meta tag by reference
+  foreach ($head_elements as $tagKey => $tag) {
+    if (isset($tag['#attributes']['name']) && $tag['#attributes']['name'] === 'viewport') {
+      $viewportTagValue = &$head_elements[$tagKey]['#attributes']['content'];
+    }
+  }
+
+  // Filter out "maximum-scale" property if exists
+  $rules = array_filter(preg_split('/\s*,\s*/', $viewportTagValue), function ($rule) {
+    return !preg_match('/^maximum-scale\s*=/i', $rule);
+  });
+
+  // Push new "maximum-scale" property
+  array_push($rules, 'maximum-scale=1');
+
+  // Set updated rules to the tag
+  $viewportTagValue = implode(', ', $rules);
```

Comment:
you don't return `$viewportTagValue` or assign it somewhere? what's the purpose of this variable then?

Reply:
Ah... It is a reference.

https://www.phpreferencebook.com/samples/php-pass-by-reference/

```php
$viewportTagValue = &$head_elements[$tagKey]['#attributes']['content'];

$viewportTagValue = 1;
// would equal to
$head_elements[$tagKey]['#attributes']['content'] = 1;
```

Reply:
Silly me, didn't see it.